internal/transformers/commonjsmodule.go

```diff
@@ -38,7 +38,7 @@ func NewCommonJSModuleTransformer(emitContext *printer.EmitContext, compilerOpti
 	tx.assignmentPatternVisitor = emitContext.NewNodeVisitor(tx.visitAssignmentPattern)
 	tx.languageVersion = compilerOptions.GetEmitScriptTarget()
 	tx.moduleKind = compilerOptions.GetEmitModuleKind()
-	return tx.newTransformer(tx.visit, emitContext)
+	return tx.NewTransformer(tx.visit, emitContext)
```

Comment:
Is this just because the declaration emitter is not in this same package?

Reply:
Yeah, but defining transforms outside the core `transform` package could also be done in some quickfixes and would be done for some lint rule (fixes) - these methods on the base transform should be exported, imo, to support defining transforms where's they're needed, rather than only in one place.

Reply:
Yeah, I think declaration emit being "different" compared to the other transforms seems fine. It really does do something different than downleveling transforms.